core/scripts/bareos-config.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/sh
```

Comment:
Sorry, but /usr/bin/sh is Red Hat / Fedora specific. In POSIX the path to the standard shell is /bin/sh. This change would break everything, but Red Hat / Fedora.
You can provide a patch that applies this change for Fedora/Red Hat only or you can change the specfile to ignore silence the warning using `__brp_mangle_shebangs_exclude`.